lib/config/presets.ts

```diff
@@ -94,7 +94,7 @@ export function parsePreset(input: string): ParsedPreset {
     }
   } else {
     // non-scoped namespace
-    [, packageName] = str.match(/(.*?)(:|$)/);
+    [, packageName] = /(.*?)(:|$)/.exec(str);
```

Comment:
I feel like our existing code was better, but I guess this new rule wants us to have the regex first?

Reply:
yes, because this is faster.
https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/prefer-regexp-exec.md